modules/itc/src/main/scala/lucuma/itc/SignificantFigures.scala

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) 2016-2022 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package lucuma.itc
+
+import eu.timepit.refined.types.numeric.PosInt
+import io.circe._
+import io.circe.refined._
+
+import scala.math._
+
+case class SignificantFigures(xAxis: Option[PosInt], yAxis: Option[PosInt])
```

Comment:
final?

Reply:
I learn we don't really need `final` anymore
https://medium.com/scala-3/opaque-type-aliases-and-open-classes-13076a6c07e4